### ISSUE TITLE
UI enhancements

### ISSUE THREAD

**Issue by ghost** [gorhill/uBlock#703]
I have a few points that could improve the UI:
1. The popup ui seems to use fonts for the symbols. I have blocked my browser to load fonts, so I only see placehoders there. Maybe you could use images instead?
2. I would like to see tooltips in the popup ui. For example that the exclamation mark tells me, that it enables/disables the strict blocking mode.
3. After changing a dynamic filtering role, I would like to have an reset option.

Thanks for your work!

**Comment by harshanvn** [comment]
> 1. The popup ui seems to use fonts for the symbols. I have blocked my browser to load fonts, so I only see placehoders there. Maybe you could use images instead?

i believe this will not be an issue with upcoming firefox anymore. new firefox will allow extension based fonts to be used even if you disable 3rd party fonts. IMHO, this is how it supposed to be in the first place.
You also, have another option. That is to use block fonts through uBlock Origin rather the browser. Which will also mitigates this issue.

> 3. After changing a dynamic filtering role, I would like to have an reset option.

Unless, i am mistaken, we have have this. After you change any rule, pad lock & clear buttons will come..

**Comment by johnnyWalker123** [comment]
Thanks for your quick reply.

1. I have changed some firefox settings (for privacy reasons). Two of
these are browser.display.use_document_fonts which I set to 0 and
gfx.downloadable_fonts.enabled which I set to false. This does more than
the blocking function in uBlock - checked this on http://ip-check.info
As a result of these settings, the popup ui isn't displayed properly
(see attached screenshot).

3. Ah ok, sorry. I havn't recognized them.


harshanvn:
